common/postAcceptanceForm.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { ReactNode } from 'react';
+import { makeCheckbox, makeShortText, makeDropdown, makeLongText, makeSection } from './questions';
+import { QuestionDefinition, QuestionSection } from './types';
+
+export const PostAcceptanceForm: Array<QuestionSection | QuestionDefinition> = [
+  makeSection(
+    <>Acceptance Form</>,
+    <div>
+      <p>
+        Congrats on being accepted to HackBeanpot! We're so excited to see you soon. Please answer
+        the following questions to help us finalize logistics of making the event as great as it can
+        be for our attendees!
+      </p>
+      *NOTE: We will only be able to guarantee swag for the first 200 hackers who complete this form
+      and qualify for domestic shipping. Thanks for your understanding.
+    </div>
+  ),
+  makeShortText(
+    'email',
+    <div>
+      <p>Email address</p>
+      Please use the email address you applied to HackBeanpot with - this should be the one where
+      you received your acceptance email!
+    </div>,
+    true,
+    'Email address'
+  ),
+  makeShortText('name', 'Full name', true, 'First Last'),
+  makeDropdown(
+    'attending',
+    'Please select if you will be attending HackBeanpot on the weekend of February 11 - 13th, 2022. If you cannot make it, please select No and leave the rest of the form blank so that we can admit others on the waitlist instead.',
+    ['Yes', 'No'],
+    true,
+    'Yes'
+  ),
```

Comment:
I don't think this question is follow-able if their answer is no, because the questions on the rest of the form are required – maybe instead just add a disclaimer section at the top 'only fill out this form if you are attending"?